modules/rosa-sdpolicy-security.adoc

```diff
@@ -58,3 +57,13 @@ See Understanding process and security for ROSA for the latest compliance inform
 [id="rosa-sdpolicy-network-security_{context}"]
 == Network security
 With {product-title}, AWS provides a standard DDoS protection on all load balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing load balancers used for {product-title}. A 10-second timeout is added for HTTP requests coming to the `haproxy` router to receive a response or the connection is closed to provide additional protection.
+
+[id="rosa-sdpolicy-etcd-encryption_{context}"]
+== etcd encryption
+
+In {product-title}, the control plane storage is encrypted at rest by default and this includes encryption of the etcd volumes. This storage-level encryption is provided through the storage layer of the cloud provider. In addition to this, you can opt to enable etcd encryption, which encrypts the key values in etcd state. The etcd encryption feature is not enabled by default and it can be enabled only at cluster installation time.
```

Comment:
```suggestion
In {product-title}, the control plane storage is encrypted at rest by default and this includes encryption of the etcd volumes. This storage-level encryption is provided through the storage layer of the cloud provider. You can also enable etcd encryption, which encrypts the key values in etcd state. The etcd encryption feature is not enabled by default and it can be enabled only at cluster installation time.
```

Reply:
@jeana-redhat thank you for your review and for this suggestion. This is now resolved.

Reply:
I have resolved this in both files.